README.rst

```diff
@@ -330,10 +330,15 @@ using OpenStack virtual machines.
 Prerequisites
 -------------
 
-An OpenStack tenant with access to the nova and cinder API (for
-instance http://entercloudsuite.com/). If the cinder API is not
-available (for instance https://www.ovh.com/fr/cloud/), some jobs
-won't run because they expect volumes attached to each instance.
+An OpenStack tenant with access to the nova and cinder API. If the
+cinder API is not available, some jobs won't run because they expect
+volumes attached to each instance.
+
+Apply for an OpenStack account
+------------------------------
+
+* Send a mail to `Loic Dachary <mailto:loic@dachary.org>`_
```

Comment:
This is a temporary measure, until a more permanent solution is found. I forwarded you the ceph-devel mail that explains it with a bit more details (title "Enabling contributors to run teuthology suites")